medusa/providers/torrent/html/hdspace.py

```diff
@@ -156,7 +157,7 @@ def parse(self, data, mode):
                                        (title, seeders), logger.DEBUG)
                         continue
 
-                    torrent_size = re.match(r'.*?([0-9]+,?\.?[0-9]* [KkMmGg]+[Bb]+).*', str(row), re.DOTALL).group(1)
+                    torrent_size = row.find('td', class_='lista222', attrs={'width': '100%'}).get_text()
```

Comment:
Yes, it will. Regex shouldn't be used "for the fun of it", but only where it's really needed. In this case the original author didn't know how to get the size without using regex, that has been fixed now.